### ISSUE TITLE
train_vq log

### ISSUE THREAD

**Issue by RohollahHS** [exitudio/MMM#10]
Great work. Thanks for sharing the code.
Could you please share the log for training the VQ-VAE?
Thanks

**Comment by exitudio** [comment]
Hi, thank you for your interest.
I attached the logs here
[VQVAE_HML3D_run.log](https://github.com/user-attachments/files/15776376/VQVAE_HML3D_run.log)
[VQVAE_KIT_run.log](https://github.com/user-attachments/files/15776377/VQVAE_KIT_run.log)

**Comment by RohollahHS** [comment]
> Hi, thank you for your interest. I attached the logs here [VQVAE_HML3D_run.log](https://github.com/user-attachments/files/15776376/VQVAE_HML3D_run.log) [VQVAE_KIT_run.log](https://github.com/user-attachments/files/15776377/VQVAE_KIT_run.log)



> Hi, thank you for your interest. I attached the logs here [VQVAE_HML3D_run.log](https://github.com/user-attachments/files/15776376/VQVAE_HML3D_run.log) [VQVAE_KIT_run.log](https://github.com/user-attachments/files/15776377/VQVAE_KIT_run.log)

Thanks. In the paper you said that you reset the codebook every 20 iterations to prevent codebook collapse. However, in the training loop https://github.com/exitudio/MMM/blob/68d850095a0640405c0d1deb1fc101f11b386736/train_vq.py#L156-L193, I did not find any thing regarding reseting codebook. Could you please clarify this?

**Comment by RohollahHS** [comment]
Thanks for sharing. Could you please explain why the FID metric in these two log files looks random? There is no sign of continuous decrease in FID, and I think it's a weird. For example, in VQVAE_KIT_run.log, it always fluctuates around 0.5 for 300,000 iterations.
